packages/components/src/TCRUserDashboard/DashboardNewsroom.tsx

```diff
@@ -155,55 +166,96 @@ const DashboardNewsroomBase: React.FunctionComponent<DashboardNewsroomProps> = p
         />
       </StyledDashboardNewsroomSection>
 
-      <FeatureFlag feature={"pew"}>
-        <Query<any>
-          query={CHANNEL_ID_FROM_NEWSROOM_ADDRESS_QUERY}
-          variables={{ contractAddress: props.newsroomAddress }}
-        >
-          {({ loading, error, data }) => {
-            if (loading || error) {
-              return <></>;
-            }
-            return (
-              <StyledDashboardNewsroomSection>
-                <StyledDashboardNewsroomHdr>Pulse Story Feed</StyledDashboardNewsroomHdr>
-                <DashboardNewsroomSubmitLink channelID={data.channelsGetByNewsroomAddress.id} />
-              </StyledDashboardNewsroomSection>
-            );
-          }}
-        </Query>
-      </FeatureFlag>
-
       <StyledDashboardNewsroomSection>
         <StyledDashboardNewsroomHdr>Boosts</StyledDashboardNewsroomHdr>
+        <StyledDashboardNewsroomSubHdr>Story Boosts</StyledDashboardNewsroomSubHdr>
+        <p>
+          Story Boosts allow supporters to give you small, incremental proceeds using ETH or credit cards. When you add
+          the Story Boost embed, your stories will be included on the Civil storyfeed and you will be able to collect
+          proceeds from the feed as well as your own site.
+        </p>
         <p>
-          Connect with the Civil community eager to fund your newsroom projects.{" "}
+          Place the following code in your CMS where you'd like a Story Boost to be displayed on your site. You can add
+          it to each story or your CMS template.
```

Comment:
@tobek anything else we should add to these embed instructions?

Reply:
Nope that's it!